### PR TITLE
[webui] Repository path editing

### PR DESCRIPTION
**Pull request by hiberis**
fixes: https://github.com/openSUSE/open-build-service/issues/923

### CODE REVIEW

File: src/api/app/controllers/webui/project_controller.rb [hunk 1]
```diff
@@ -894,19 +894,20 @@ def remove_path_from_target
     redirect_to :action => :repositories, :project => @project
   end
 
-  def move_repo(direction)
+  def move_path_up
```
Comment (hennevogel) [inline src/api/app/controllers/webui/project_controller.rb]: Why do you duplicate this? [Don't repeat yourself!](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

**Comment by hennevogel** [comment]
Other then that. Awesome, thanks :-)

File: src/api/app/controllers/webui/project_controller.rb (hunk 1 above)
Comment (jspanjers) [inline src/api/app/controllers/webui/project_controller.rb]: In this case, I thought that removing the duplication causes the code to be harder to understand. But I guess it does not need to be. I can give it a try cleaning it up.

Comment (hiberis) [inline src/api/app/controllers/webui/project_controller.rb]: In this case, I thought that removing the duplication causes the code to be harder to understand. But I guess it does not need to be. I can give it a try cleaning it up.

Comment (hennevogel) [inline src/api/app/controllers/webui/project_controller.rb]: Yeah I rather fix one occurrence of this then two :-) Thank you so much!

**Pull request merged**